### PR TITLE
Add basic sponsors list to landing page

### PR DESCRIPTION
**Pull request by johnmccabe**
This is a basic implementation as we do not have a formal design at this point.

- sponsors should be added to the appropriate tier in _data/sponsors
- logos should be added to images/sponsors
- asyncy logo is a placeholder

### Desktop

screen shot 2019-03-01 at 02 05 20

### Mobile
screen shot 2019-03-01 at 02 05 30

### CODE REVIEW

**Comment by johnmccabe** [comment]
derek assign: alexellis

**Comment by johnmccabe** [comment]
https://raw.githubusercontent.com/openfaas/faas/master/.DEREK.yml

**Comment by johnmccabe** [comment]
Derek assign: alexellis

**Comment by alexellis** [comment]
👍 thank you.

I would say these need to go above the copyright / company notice.

**Comment by johnmccabe** [comment]
## Desktop
screen shot 2019-03-01 at 10 42 53

## Mobile
screen shot 2019-03-01 at 10 43 06

**Comment by johnmccabe** [comment]
@alexellis did you ever get a formal logo from @stevepeak.

Ideally if we could get something with the name and logo it would be preferable (can then drop the text name).

Preferably something like this.

screen shot 2019-03-01 at 10 45 36

We can tweak/hone this over the coming days/weeks.

**Comment by stevepeak** [comment]
@johnmccabe I can provide such a logo in this rectangular dimensions today. Thank you!

**Comment by stevepeak** [comment]
asyncy-artboard

Here we go 👍 🙇

**Comment by johnmccabe** [comment]
@alexellis thats @garden-io added as well, can you reach out to them (or tag someone into this thread) to check if the logo, alt text is ok (link is to https://garden.io/, alt/title text is from their twitter handle - "Garden - Next-generation development platform for backend systems")

@stevepeak the alt/title text attributes on the logo are set to "ASYNCY - The Kubernetes Native DSL", let me know if you need that tweaked - link is to https://asyncy.com/

**Comment by johnmccabe** [comment]
screen shot 2019-03-01 at 23 17 18

**Comment by alexellis** [comment]
Pinging @edvald from Garden

Thanks @johnmccabe

**Comment by johnmccabe** [comment]
@alexellis I can remove the Garden entry so you can just get this merged if you wish

**Comment by alexellis** [comment]
Thank you John

File: _includes/sponsors.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<section class="tiers">
+    <div class="text-container">
+        {% for tier in site.data.sponsors %}
+        {% if tier[1].size > 0 %}
+        <div class="tier">
+            <div class="title">{{tier[0] | capitalize}} Sponsors</div>
+            <div class="sponsors">
+                {% for sponsor in tier[1] %}
+                {% include sponsor.html name=sponsor.name logo=sponsor.logo alt=sponsor.alt %}
```
Comment (stevepeak) [inline _includes/sponsors.html]: @johnmccabe Looks like the links are pointing at `href="#"` may I suggest to point them to each vendor and include a `?utm_campaign=openfaas` for example.

Comment (alexellis) [reply _includes/sponsors.html]: I've fixed the URL, thanks for pointing that out.

Apparently `utm_*` needs a whole bunch of required parameters i.e. http://blog.rafflecopter.com/2014/04/utm-parameters-best-practices/

**Comment by alexellis** [comment]
@johnmccabe I've commented out Garden due to the pledge being changed after the payment cycle and this meant that their tier for March is still as an individual, not a sponsor level. We can enable after the April billing cycle.

**Pull request merged**